checkpoint.go

```diff
@@ -18,11 +18,15 @@ var checkpointCommand = cli.Command{
 		cli.StringFlag{Name: "image-path", Value: "", Usage: "path for saving criu image files"},
 		cli.StringFlag{Name: "work-path", Value: "", Usage: "path for saving work files and logs"},
 		cli.BoolFlag{Name: "leave-running", Usage: "leave the process running after checkpointing"},
-		cli.BoolFlag{Name: "tcp-established", Usage: "allow open tcp connections"},
-		cli.BoolFlag{Name: "ext-unix-sk", Usage: "allow external unix sockets"},
-		cli.BoolFlag{Name: "shell-job", Usage: "allow shell jobs"},
+		cli.BoolFlag{Name: "no-tcp", Usage: "fail on open tcp connections"},
+		cli.BoolFlag{Name: "no-ext-unix-sk", Usage: "fail on external unix sockets"},
+		cli.BoolFlag{Name: "no-shell-job", Usage: "fail on shell jobs"},
+		cli.BoolFlag{Name: "no-file-locks", Usage: "fail on file locks, for safety"},
```

Comment:
For me I guess the question is about what are the reasonable defaults. I think most people will want tcp-established enabled, and many people will want file-locks enabled. It's unfortunate to have to pass several flags for the most common operations. @crosbymichael what are your thoughts?

(The shell job one needs to be reverted though, as mentioned above)